solver/src/main/java/org/chocosolver/solver/constraints/real/RealPowerPropagator.java

```diff
@@ -0,0 +1,108 @@
+package org.chocosolver.solver.constraints.real;
+
+import org.chocosolver.solver.constraints.Propagator;
+import org.chocosolver.solver.constraints.PropagatorPriority;
+import org.chocosolver.solver.exception.ContradictionException;
+import org.chocosolver.solver.expression.continuous.arithmetic.RealIntervalConstant;
+import org.chocosolver.solver.variables.RealVar;
+import org.chocosolver.solver.variables.events.RealEventType;
+import org.chocosolver.util.ESat;
+import org.chocosolver.util.objects.RealInterval;
+import org.chocosolver.util.tools.RealUtils;
+
+/**
+ * It's a pure java alternative to ibex that must be used to calculate
+ * power operations (y = x^k) with integer positive exponents (k).
+ *
+ * <p>

```

Comment:
The 3 following lines are out-dated, there is no reference to `jinterval` anymore, if I'm correct.